bucket/manifold-viewer-9-official.json

```diff
@@ -0,0 +1,20 @@
+{
+    "version": "9.0.174",
+    "description": "Official build of Manifold Viewer Release 9",
+    "homepage": "http://manifold.net/",
+    "license": "Unknown",
+    "depends": "extras/vcredist2017" ,
+    "url": "http://www.manifoldgis.com/updates/manifold-viewer-9.0.174-x64.zip",
+    "extract_dir": "manifold-viewer-9.0.174-x64",
+    "hash": "sha256:23e85f8b1320c161ef6288f65c1ae56f5098dabbe2d9d98814c7f6dd085843b4",
```

Comment:
Exchange position of these two properties

Reply:
ok.